float64 tunningParam1
float64 tunningParam2
float64 tunningParam3
float64 tunningParam4
float64 tunningParam5
float64 tunningParam6
float64 tunningParam7
float64 tunningParam8
float64 tunningParam9
